CMake/FindNGHTTP2.cmake

```diff
@@ -0,0 +1,18 @@
+include(FindPackageHandleStandardArgs)
+
+find_path(NGHTTP2_INCLUDE_DIR "nghttp2/nghttp2.h")
+
+find_library(NGHTTP2_LIBRARY NAMES nghttp2 libnghttp2)

```

Comment:
CMake does automatically add a `lib` prefix when searching on platforms that use it.  This can be just

```cmake
find_library(NGHTTP2_LIBRARY NAMES nghttp2)
```